examples/structured_2d_dgsem/elixir_euler_warm_bubble.jl

```diff
@@ -0,0 +1,123 @@
+using OrdinaryDiffEq
+using Trixi
+
+# Physical constants
+g::Float64 = 9.81          # gravity of earth
+p_0::Float64 = 100_000.0   # reference pressure
+c_p::Float64 = 1004.0      # heat capacity for constant pressure (dry air)
+c_v::Float64 = 717.0       # heat capacity for constant volume (dry air)
+R = c_p - c_v              # gas constant (dry air)
+gamma = c_p / c_v          # heat capacity ratio (dry air)
```

Comment:
These globals are not typed. Thus, there will be a type instability every time you access them.

Reply:
Do you actually need all of these globals?

Reply:
Thanks for pointing this out! I though it would be inferred from the typed c_p and c_v.

Reply:
I need the constants for this particular elixir. But not all them would have to be global. Would it be better to move those?

Reply:
Yes, their types are inferred - but they are just global variables (neither constants not typed globals).
In general, it's best to avoid global variables.

Reply:
Ok, thanks!